python/cuml/linear_model/linear_regression.pyx

```diff
@@ -316,6 +375,61 @@ class LinearRegression(Base,
 
         return self
 
+    def _fit_multi_target(self, X, y, convert_dtype=True, sample_weight=None):
+        # In the cuml C++ layer, there is no support yet for multi-target
+        # regression, i.e., a y vector with multiple columns.
+        # We implement the regression in Python here.
```

Comment:
What is the effect on performance? Were you able to benchmark the code? I assume that users would expect a multi-target approach to actually be faster compared to running the regression problems independently.

Reply:
Good question. It is roughly 20-50x faster on a representative use case. I have added a comment to the PR to provide the proper motivation for this PR.